Dockerfile

```diff
@@ -1 +1,25 @@
+ARG GOLANG_VERSION=1.19
+FROM golang:${GOLANG_VERSION} as cli-builder
+WORKDIR /go/src/github.com/mendersoftware/mender-cli
+ARG MENDER_CLI_VERSION=master
+RUN git clone -b $MENDER_CLI_VERSION https://github.com/mendersoftware/mender-cli.git . && \
+    make get-deps && \
+    make build
+
+FROM golang:${GOLANG_VERSION} as artifact-builder
+WORKDIR /go/src/github.com/mendersoftware/mender-artifact
+ARG MENDER_ARTIFACT_VERSION=master
```

Comment:
This should be defined only once (DRY principle).

Reply:
Yeah. It's leftover from local testing. Cleaning up.

Reply:
Actually it doesn't work without this ARG...

Reply:
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact <- FYI explains this (quite poorly IMO), but you need to put a `ARG <variableName>` (in this particular case `ARG MENDER_ARTIFACT_VERSION`) in every stage you want to use the passed argument...

Reply:
@mzedel will moving all `ARG VAR=default` on the top of the file and declaring `ARG VAR` in each of the build sections look better?